crossing-training.lic

```diff
@@ -679,18 +685,18 @@ class CrossingTraining
 
   def climb_branch
     start_script('skill-recorder') unless Script.running?('skill-recorder')
-    walk_to(2245)
+    walk_to @climbing_target['id']
     check_research
     if DRSkill.getxp('Athletics') < 29
-      walk_to(2245)
+      walk_to @climbing_target['id']
       if UserVars.athletics < 550
```

Comment:
This check is against 550 athletics, but the comment in `base-athletics` says undergola_branch is for 530, and the `if` in `train_athletics` is against 400. Which one is correct?